### PR TITLE
fix: Make login and plugin commands visible in CLI help text

### PR DESCRIPTION
**Pull request by hermanschaaf**

### CODE REVIEW

File: cli/cmd/login.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func newCmdLogin() *cobra.Command {
 		Short:   loginShort,
 		Long:    loginLong,
 		Example: loginExample,
-		Hidden:  true,
+		Hidden:  false,
```
Comment (disq) [inline cli/cmd/login.go:48]:
```suggestion
```

File: cli/cmd/logout.go [hunk 2]
```diff
@@ -20,7 +21,7 @@ func newCmdLogout() *cobra.Command {
 	loginCmd := &cobra.Command{
 		Use:    "logout",
 		Short:  logoutShort,
-		Hidden: true,
+		Hidden: false,
```
Comment (disq) [inline cli/cmd/logout.go:24]:
```suggestion
```

File: cli/cmd/plugin_publish.go [hunk 3]
```diff
@@ -35,7 +35,7 @@ func newCmdPluginPublish() *cobra.Command {
 		Short:   pluginPublishShort,
 		Long:    pluginPublishLong,
 		Example: pluginPublishExample,
-		Hidden:  true,
+		Hidden:  false,
```
Comment (disq) [inline cli/cmd/plugin_publish.go:38]:
```suggestion
```

File: cli/cmd/switch.go [hunk 4]
```diff
@@ -28,7 +28,7 @@ func newCmdSwitch() *cobra.Command {
 		Long:    switchLong,
 		Example: switchExample,
 		Args:    cobra.MaximumNArgs(1),
-		Hidden:  true,
+		Hidden:  false,
```
Comment (disq) [inline cli/cmd/switch.go:31]:
```suggestion
```

**Review by erezrokah — APPROVED** [review]

**Review by disq — APPROVED** [review]

**Comment by disq** [comment]
Test golden needs more items now that we have more docs

**Pull request merged**